robottelo/hosts.py

```diff
@@ -253,6 +254,19 @@ def download_install_rpm(self, repo_url, package_name):
         if result.status != 0:
             raise ContentHostError(f'Failed to install {package_name} rpm.')
 
+    def download_repos(self, repo_name):
+        """Downloads the satellite or capsule repos on the machine
+        :param repo_name: satellite or capsule repo_name
+        """
+        repo_location = (
+            f'{settings.repos.ohsnap_repo_host}/api/releases/'
+            f'{settings.server.version}/el7/{repo_name}/repo_file'
```

Comment:
I think this should be `self.satellite.version` else the destructive sat spined up with different rhel version than the xDist Sats rhel version then this will fail or return a wrong value.

Though there might be less possibility of spinning different rhel vers sat than xDist sats rhel version, but we never know.